[DOCS] Updating instructions for installing EMT on Virtual Machines (#285)
open-edge-platform/edge-microvisor-toolkit: docs/developer-guide/get-started/deployment/emt-vm-guest.md

## Hyper-V

> When using Hyper-V, you can install the ISO to a virtual hard drive that you create.
> See the steps below:

Reviewer: If now there's only one option, isn't this statement too obvious?
In virtual machine, you can use a virtual hard drive... That you create, or someone else creates, doesn't matter :)
```suggestion
To install the ISO on a fresh virtual hard drive:
```
